docs/guides/jsdom.md

```diff
@@ -11,6 +11,31 @@ gets run *before* React's code is run.
 
 As a result, a standalone script like the one below is generally a good approach:
 
+`jsdom v10~`:
+
+```js
+/* setup.js */
+
+var { JSDOM } = require('jsdom');
+var jsdom = new JSDOM('');
+
+global.document = jsdom;
```

Comment:
Thank you for reviewing. 

I fixed the doc changes with https://github.com/airbnb/enzyme/pull/921/commits/35c2f5e29e9f942975b5fc0838b2838bbd0c90de

I created a working sample repository with `jsdom ^10.1.0`.
https://github.com/kenju/enzyme-draftjs-sample/blob/master/test/setup.js